src/store/api/weatherProviders/climacell.js

```diff
@@ -0,0 +1,150 @@
+import { isToday, isTomorrow, startOfDay } from 'date-fns';
+import getAPIData from '../apiutils';
+import * as utils from './weatherutils';
+
+const API_KEY = process.env.VUE_APP_CLIMACELL_API_KEY;
+const BASE_URL = 'https://api.climacell.co/v3/weather/forecast/';
+
+const getImageNumber = (weatherCode, isDayTime) => {
+  const weatherCodeTranslatorDay = {
+    freezing_rain_heavy: '26',
```

Comment:
maybe it's better to rename icons instead? 
As for me having `snow_light.svg` is far more obvious than `22.png`

Reply:
I use this icons in current weather too, where API returns a icon number.